code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -13,6 +13,28 @@
 	holder.remove_reagent(id, total_depletion_rate) //medicine reagents stay longer if you have a better metabolism
 	return STATUS_UPDATE_NONE
 
+/datum/reagent/medicine/proc/heal_external_limb(obj/item/organ/external/organ, volume)
+	return
+
+/datum/reagent/medicine/proc/splash_human(mob/living/carbon/human/H, volume)
+	var/turf_volume = max(0, (volume - 100) / 3)
+	volume = volume - turf_volume
+
+	var/volume_per_bodypart = volume / H.bodyparts.len
+
+	for(var/obj/item/organ/external/organ in H.bodyparts)
+		if(!get_location_accessible(H, organ.limb_name))
+			turf_volume += volume_per_bodypart
+			continue
+
+		heal_external_limb(organ, volume_per_bodypart)
+
+	H.updatehealth(reason = "[id] splashed")
+	H.UpdateDamageIcon()
+
+	if(turf_volume > 0)
+		reaction_turf(get_turf(H), turf_volume)
+
```

Comment:
I feel like these should be specific to some parent subtype of just Styptic Powder/Silver Sulfadiazine/Synthflesh? If no other medicine chems use this, there's no point to having a noop proc defined and ran on all of them when splashing

Reply:
Actually, scratch that, since defining this exclusively on those 3 chems would mean every other chemical would just apply the full volume when splashed, which would be a weird inconsistency - in that case, why isn't this volume spread behavior defined on splash reactions in general, rather than only applying to medicine reagents?